crypto/s2n_ecc_evp.c

```diff
@@ -185,6 +191,31 @@ static int s2n_ecc_evp_compute_shared_secret(EVP_PKEY *own_key, EVP_PKEY *peer_p
     return 0;
 }
 
+static bool s2n_ecc_evp_is_available(uint16_t query_iana_id) {
```

Comment:
Side note: I find the naming of this file a little confusing. We use EVP APIs even in blocks of code that aren't guarded by EVP_APIS_SUPPORTED. More of a question to the group if we want to keep perpetuating the s2n_ecc_evp_* prefix for all functions in this file. If so, we can just rename this particular function to something curve related and continue with this prefix. The reason I started thinking about this is because this function doesn't really check anything evp related for 256/384/521 curves